packages/block-editor/src/components/rich-text/index.js

```diff
@@ -824,51 +820,44 @@ export class RichText extends Component {
 		} );
 	}
 
-	/**
-	 * Splits the content at the location of the selection.
-	 *
-	 * Replaces the content of the editor inside this element with the contents
-	 * before the selection. Sends the elements after the selection to the `onSplit`
-	 * handler.
-	 *
-	 * @param {Array}  blocks  The blocks to add after the split point.
-	 * @param {Object} context The context for splitting.
-	 */
-	splitContent( blocks = [], context = {} ) {
-		if ( ! this.onSplit ) {
+	onSplit( record, pastedBlocks = [] ) {
+		const { onReplace, onSplit, onSplitMiddle } = this.props;
```

Comment:
onSplitMiddle is not clear to me and I think it's not documented right?

Reply:
Is this something to be marked unstable?

Reply:
Seems good to mark as unstable for now. Will adjust.

Reply:
Done in 199a1e9.